brainspell/test_tornado.py

```diff
@@ -131,28 +134,51 @@ def test_procfile():
         assert filename in os.listdir()
 
 
-""" TODO: get selenium testing working
-def test_existence(): #Using selenium testing
+
+def test_existence(): #Using selenium testing to verify existence of site elements
     driver.get("https://brainspell.herokuapp.com")
-    assert "Brainspell" in driver.title #Checks website title is accurate
+    driver.implicitly_wait(3)
+    assert "Brainspell" in driver.title #Checks website was correctly received
```

Comment:
Check for the HTTP status code instead, rather than checking the title. The title might change.